osu.Game/Screens/Select/PlaySongSelect.cs

```diff
@@ -20,6 +24,8 @@ public class PlaySongSelect : SongSelect
         private OsuScreen player;
         private readonly ModSelectOverlay modSelect;
         private readonly BeatmapDetailArea beatmapDetails;
+        private IEnumerable<Mod> originalMods;
+        private bool controlPressed;

```

Comment:
Would rather store the inputmanager from load() and checking inputManager.CurrentState.Keyboard.ControlPressed.